products/compute/terraform.yaml

```diff
@@ -2500,6 +2500,40 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       pathMatchers.pathRules.urlRedirect.stripQuery: !ruby/object:Overrides::Terraform::PropertyOverride
         required: true
         description: '{{description}} This field is required to ensure an empty block is not set. The normal default value is false.'
+      pathMatchers.defaultRouteAction.weightedBackendServices.weight: !ruby/object:Overrides::Terraform::PropertyOverride
+        validation: !ruby/object:Provider::Terraform::Validation
+          function: 'validation.IntBetween(0, 1000)'
```

Comment:
should we add send_empty values? I'm guessing that if we do add it, it will always send zero which is not really what we want, but I don't think we can unweigh things then. Might just be a limitation we have to live with, backendService is the same way.

Reply:
i tried adding them in here, but i get an error when `defaultService` is set because they're conflicting, so we might just have to work with it like this.